extra/release.py

```diff
@@ -3,6 +3,8 @@
 
 """A utility script for automating the beets release process.
 """
+from __future__ import absolute_import
+from __future__ import print_function
```

Comment:
FWIW, this release program is actually a pure Python 3 program. I'm not even sure it works under Python 2.